tests/python/unittest/test_gluon.py

```diff
@@ -2587,6 +2587,21 @@ def hybrid_forward(self, F, x):
             net = Net(act0, act1, shape, slice)
             check_layer_forward_withinput(net, x)
 
+
+@with_seed()
+def test_groupnorm():
+    ctx=mx.context.current_context()
+    x = mx.nd.random.uniform(1, 2, (4, 16, 8, 8), ctx=ctx)
+    gn = mx.gluon.nn.GroupNorm(4, 16)
+    gn.initialize(ctx=ctx)
+    y = gn(x)
+    y = y.reshape(0, 4, -1)
+    print('y.mean(2) =', y.mean(2))
```

Comment:
can we use logging.info instead of a print statement here?

Reply:
thanks for pointing out :)